Tutorial.md

```diff
@@ -11,477 +12,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
-
-> **Note:** Should you need a clarification of some terms as used through this document refer to [API Blueprint Glossary of Terms][].
-
-#### API Name
-
-    # Gist Fox API
-
-Every good API should have a name. So does ours – *"Gist Fox API"*. The first Markdown header in our document represents just that – the API name.
-
-#### API Description
-
-    Gist Fox API is a **pastes service** similar to [GitHub's Gist][http://gist.github.com].
-
-The API Name might be followed by any arbitrary Markdown-formatted discussion. Preferably about your API.
-
-> **Note:** We will add much richer and more helpful discussion in the final version of this blueprint. Adding details about authentication, used media types and error handling to API description is always a great idea!
->
-> **Note:** For the sake of simplicity we will now omit user management and authentication. These features will be discussed in a future instalment of the API Blueprint Tutorial.
-
-## Markdown
-All you really need to write a blueprint is a text editor. A Markdown editor is even better. Anything you like. From Vi to Byword. Online editors are great too! You can even directly edit a blueprint on GitHub in the repository your service is living!
-
-If you are completely new to Markdown it is now the best time to learn about it. Just head over to [Markdown Tutorial][] and enjoy the ride. Just don't forget to come back we have some cool stuff too!
-
-Should you just need to refresh Markdown syntax you can always check the one and only [Gruber's Original][] documentation.
-
->  **Note:** It is not essentially required to know the Markdown syntax but it will help you a lot. The knowledge of Markdown will make some API Blueprint constructs more intuitive.
-
-## First Resource
-The first resource in our API is its root. The entry point to our API is defined like so:
+## API Blueprint
 
-    # Gist Fox API Root [/]
-    Gist Fox API entry point.
+The start of the Blueprint is going to look as follows:
 
-    This resource does not have any attributes. Instead it offers the initial API affordances in the form of the HTTP Link header and HAL links.
+```markdown
+FORMAT: 1A
 
-    ## Retrieve Entry Point [GET]
+# Polls
 
-    + Response 200 (application/hal+json)
-        + Headers
+Polls is a simple web service that allows consumers to view polls and vote in them.
+```
 
-                Link: <http:/api.gistfox.com/>;rel="self",<http:/api.gistfox.com/gists>;rel="gists"
+## Metadata
 
-        + Body
-
-                {
-                    "_links": {
-                        "self": { "href": "/" },
-                        "gists": { "href": "/gists?{since}", "templated": true }
-                    }
-                }
+At the very top of this Blueprint, you will find a metadata section. In this case we have specified that `FORMAT` has the value of `1A`. The format keyword is used to specify the version of the API Blueprint. In this case 1A.
```

Comment:
I would sync up your language with my [comment here](https://github.com/apiaryio/api-blueprint/pull/154/files#r25521731) about "starting" or "beginning." It seems like the language should be beginning, middle, end rather than top, middle, bottom. You usually refer to writing as the former, maybe?